sklearn/utils/_unittest_backport.py

```diff
@@ -0,0 +1,173 @@
+import re
```

Comment:
This should have attribution to where it is backported from (and the copyright notice)